scripts/RNA_analysis/make_complexity_plots.py

```diff
@@ -46,11 +48,19 @@
 #SBATCH {5}
 
 module load bioinfo-tools
+module load {6}
 module load {3}
 module load {2}
 
 cd {4}
+""".format(name, mail, preseq, bam, path, extra_arg, bed)
+
+if len(sys.argv)==6:
```

Comment:
Might be nicer to do a proper string comparison here? This will evaluate to true even if someone specifies `false` for single end...